### PR TITLE
feat(quantic): added ariaLabel for timeframeFacet.

### PR DESCRIPTION
**Pull request by MSLI-Coveo**
[SVCC-2301](https://coveord.atlassian.net/browse/SVCC-2301)
feat(quantic): added ariaLabel for timeframeFacet.
Screen Shot 2022-08-22 at 10 06 23 AM

### CODE REVIEW

**Comment by lvu285** [comment]
Can you please double check the scenario when user input incorrect format into the input box and click Apply, it should read the warning message as well

File: packages/quantic/force-app/main/default/labels/CustomLabels.labels-meta.xml [hunk 1]
```diff
@@ -497,6 +497,13 @@
         <protected>false</protected>
         <shortDescription>Apply</shortDescription>
     </labels>
+    <labels>
+        <fullName>quantic_TimeframeInputApply</fullName>
+        <value>Apply custom timeframe values for the {{0}} facet</value>
+        <language>en_US</language>
+        <protected>false</protected>
+        <shortDescription>Apply custom timeframe values for the [component label] facet</shortDescription>
```
Comment (lbergeron) [inline packages/quantic/force-app/main/default/labels/CustomLabels.labels-meta.xml:505]: Be careful with the length of the `shortDescription` value. It is limited to something like 80 characters. You won't be able to deploy the label if the `shortDescription` is too long.

**Review by lbergeron — APPROVED** [review]
Looks good to me!

File: packages/quantic/force-app/main/default/labels/CustomLabels.labels-meta.xml (hunk 1 above)
Comment (erocheleau) [inline packages/quantic/force-app/main/default/labels/CustomLabels.labels-meta.xml:505]: @acarpentier01 string revision here :)

**Review by erocheleau — APPROVED** [review]
String revision but otherwise, it's all okay!

File: packages/quantic/force-app/main/default/labels/CustomLabels.labels-meta.xml (hunk 1 above)
Comment (acarpentier01) [inline packages/quantic/force-app/main/default/labels/CustomLabels.labels-meta.xml:505]:
```suggestion
        <shortDescription>Apply custom time frame values for the [component label] facet.</shortDescription>
```

**Review by btaillon — APPROVED** [review]

File: packages/quantic/force-app/main/default/labels/CustomLabels.labels-meta.xml (hunk 1 above)
Comment (MSLI-Coveo) [reply packages/quantic/force-app/main/default/labels/CustomLabels.labels-meta.xml]: Thanks for info!!!

**Pull request merged**